### PR TITLE
Fixed includes/class-wc-auth.php PHPCS violations

### PR DESCRIPTION
**Pull request by claudiosanches**

### CODE REVIEW

File: includes/class-wc-auth.php [hunk 1]
```diff
@@ -327,50 +325,74 @@ protected function auth_endpoint( $route ) {
...
+
+			// Login endpoint.
+			if ( 'login' === $route && ! is_user_logged_in() ) {
+				wc_get_template(
+					'auth/form-login.php', array(
+						'app_name'     => wc_clean( $data['app_name'] ),
+						'return_url'   => add_query_arg(
+							array(
+								'success' => 0,
+								'user_id' => wc_clean( $data['user_id'] ),
```
Comment (kloon) [inline includes/class-wc-auth.php]: Instead of wc_clean should this not rather use absint?

File: includes/class-wc-auth.php [hunk 2]
```diff
@@ -327,50 +325,74 @@ protected function auth_endpoint( $route ) {
...
-				) );
+			} elseif ( 'authorize' === $route && current_user_can( 'manage_woocommerce' ) ) {
+				// Authorize endpoint.
+				wc_get_template(
+					'auth/form-grant-access.php', array(
+						'app_name'    => wc_clean( $data['app_name'] ),
+						'return_url'  => add_query_arg(
+							array(
+								'success' => 0,
+								'user_id' => wc_clean( $data['user_id'] ),
```
Comment (kloon) [inline includes/class-wc-auth.php]: Same as above, rather absint?

File: includes/class-wc-auth.php [hunk 3]
```diff
@@ -327,50 +325,74 @@ protected function auth_endpoint( $route ) {
...
+				$response      = $this->post_consumer_data( $consumer_data, $this->get_formatted_url( $data['callback_url'] ) );
 
 				if ( $response ) {
-					wp_redirect( esc_url_raw( add_query_arg( array( 'success' => 1, 'user_id' => wc_clean( $_REQUEST['user_id'] ) ), $this->get_formatted_url( $_REQUEST['return_url'] ) ) ) );
+					wp_redirect(
+						esc_url_raw(
+							add_query_arg(
+								array(
+									'success' => 1,
+									'user_id' => wc_clean( $data['user_id'] ),
```
Comment (kloon) [inline includes/class-wc-auth.php]: absint?

**Review by kloon — APPROVED** [review]
Just a couple of questions but don't think it relates to this PR since the code was originally using wc_clean. For the PHPCS changes all good!

**Comment by claudiosanches** [comment]
@kloon we can't use `absint()`, since it is not an ID on WordPress, this is a `user_id` that comes from any application, so people could send things like `app_xxxx_123`. It's used to know what user is coming back from our Auth process.

**Pull request merged**